docs/self-managed/zeebe-deployment/operations/cluster-scaling.md

```diff
@@ -117,6 +145,872 @@ The response to this request would contain a `changeId`, `currentTopology`, plan
 
 ```
 
+#### 2.b Scaling brokers and partitions
+
+:::warning
+Partition scaling is still an experimental feature, it is required to set `ZEEBE_BROKER_EXPERIMENTAL_FEATURES_ENABLEPARTITIONSCALING=true` to enable it.
+:::
+
+Run the following to send the request to the Zeebe Gateway to add 3 new brokers to the cluster and set the number of partition to 6.
+
+```
+curl -X 'PATCH' \
+   'http://localhost:9600/actuator/cluster' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+        "brokers": {
+          "add": [3,4,5]
+        },
+        "partitions": {
+          "count": 6,
+          "replicationFactor": 3
+        }
+      }'
+```
+
+If you don't intend to add new brokers to the cluster, you can skip the `"brokers"` section:
+
+```
+curl -X 'PATCH' \
+   'http://localhost:9600/actuator/cluster' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+        "partitions": {
+          "count": 6,
+          "replicationFactor": 3
+        }
+      }'
+```
+
+  <details>
+
+    <summary> Example response</summary>
+
+```json
+{
```

Comment:
🔧 I would remove the exporters, or at least trim it to just the CamundaExporter, as this should be as close to what a user would see.

It would also be good to document what to look for in the response, but that can be done later.